Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1590,9 +1590,9 @@ def get_detections_entities(detections_ids: list):
         :param detections_ids: IDs of the requested detections.
         :return: Response json of the get detection entities endpoint (detection objects)
     """
-    ids_json = {'ids': detections_ids} if LEGACY_VERSION else {"composite_ids": detections_ids}
-    url = '/detects/entities/summaries/GET/v1' if LEGACY_VERSION else '/alerts/entities/alerts/v2'
-    demisto.debug(f"Getting detections entities from {url} with {ids_json=}. {LEGACY_VERSION=}")
+    ids_json = {'ids': detections_ids}
+    url = '/detects/entities/summaries/GET/v1'
+    demisto.debug(f"Getting detections entities from {url} with {ids_json=}.")
```

Comment:
Hi @bdanjoux I'm not sure if we should remove the support for legacy version. Is there a specific issue you are experiencing?

Reply:
Hi @amshamah419 I am experiencing an issue where an integration instance is not able to fetch detection summaries, that instance is configured with "use the legacy API" unticked.

As I explain in the PR description, the API endpoint /detects/entities/summaries/GET/v1 is not deprecated, it is very much current, please check the Crowdstrike API documentation.

This problem was introduced in the commit https://github.com/demisto/content/commit/f53aa8711a013ebe4a893358dfad8f4519105ec7    back in July.

in that July PR, many commands were updated as their endpoint was being deprecated and replaced with a new one, hence the LEGACY switch.

However, the command to fetch detection summaries should not have been modified, as its underlying endpoint was not deprecated. Additionally, what that still valid endpoint was replaced with, is an endpoint for alert summaries, not detection summaries.

This is why my integration instance is failing to retrieve detection summaries.

I input a valid detection ID to the detection summary fetching command, but the integration command feeds it to an endpoint that expects an alert ID instead.

In summary, I am not removing support for the legacy API endpoint, I am only trimming an incorrectly amended integration command.

Reply:
Hi @bdanjoux Can you please open an issue with CS? I think this is something which should be investigated since it could have effects elsewhere.